packages/react-testing/src/element.ts

```diff
@@ -220,18 +222,120 @@ export class Element<Props> implements Node<Props> {
     });
   }
 
+  debug(verbosity?: number, depth?: number): string;
+  debug(options?: DebugOptions): string;
+  debug(options: number | DebugOptions = {}, depth?: number) {
```

Comment:
Why have two options here?

Reply:
/shrug i wanted to avoid having to write `debug(undefined, 0)`

Reply:
`debug({depth: 0})` is literally fewer characters than that and is not inscrutable...

Reply:
i'll 🔥 the positional params